src/theme/themeMerge/themeMerge.spec.ts

```diff
@@ -0,0 +1,47 @@
+// Utils
+import { themeMerge } from '../index';
+import { RootTheme } from '../index';
+
+// TEST SETUP
+const data = {
+    fonts: {
+        base: 'Comic Sans MS',
+    },
+    colors: {
+        primary: {
+            base: '#ff0000',
```

Comment:
Definitely will add some snapshot tests.

Someone wanting to remove an entry I think is a bigger conversation and out of the scope of this initial offering. We've been pushing hard for people _not_ to do that since they're still learning how to use theming. Maybe something to address later.

Reply:
Added a snapshot test for the `Button` as that has edge-case you defined.